Radzen.Blazor/RadzenTabs.razor.cs

```diff
@@ -179,6 +179,14 @@ protected override string GetComponentCssClass()
             {
                 positionCSS = "rz-tabview-left";
             }
+            else if(TabPosition == TabPosition.TopRight)
+            {
+                positionCSS = "rz-tabview-top-right";
```

Comment:
Let's add a CSS class instead of replacing it to reduce the duplicated code in the SCSS file. In this case it would be `positionCSS = "rz-tabview-top rz-tabview-top-right";`